lib/python/test/testsuite/Math/math.py

```diff
@@ -0,0 +1,66 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2022 PIConGPU contributors
+Authors: Mika Soren Voss
+License: GPLv3+
+
+This module includes some commonly used math functions.
+This module is not yet complete with this version.
+
+Routines in this module:
+
+growthRate(f, time, interval=None):
+    This function calculate the growthrate of a given quantity f(t)
+"""
+
+__all__ = ["growthRate"]
+
+import numpy as np
+
+
+def growthRate(f, time, interval=None):
+    """
+    This function calculate the growthrate of a given quantity f(t)
+
+    Gamma_f(t_k)= log(f(t_(k+1))/f(t_(k-1)))/t_(k+1)- t_(k-1)
+
+    input:
+    -------
+    f:         array
+                quantity whose growth rate is to be determined
+
+    time:      array
+               the array of the timesteps with the same size
+               as the quantity f
+
+    interval:  [starttime, endtime], optional
+                determines the time interval in which the growth rate
+                is to be determined
+
+    return:
+    -------
+    growth rate: array if interval = None
+                  growth rate calculated using the above formula
+
+                tuple of (growthrate, start, stop)
+                growth rate calculated using the above formula and
+                the start and end of the interval
+    """
+
+    if interval is None:
+        Gamma = 0.5 * np.log(f[3:]/f[1:-2])/(time[3:] - time[1:-2])
```

Comment:
here, you exclude `f[0]` (which is good for KHI. But please add a comment why you do that especially in the context of having a time selector available.